tools/python/boutiques/localExec.py

```diff
@@ -833,14 +833,16 @@ def escape_string(s):
                     val = in_out_dict[param_id]
                     if type(val) is list:
                         s_val = ""
+                        list_sep = (self.safeGet(param_id, 'list-separator')
+                                    or " ")
```

Comment:
I'm not sure I like the `or` here, as the empty string evaluates to `False`. I reckon having fixed a few bugs that were coming from that. I'd say `if list_sep == None: list_sep = " "` is safer.